bundles/org.openhab.binding.airparif/src/main/java/org/openhab/binding/airparif/internal/handler/AirParifBridgeHandler.java

```diff
@@ -0,0 +1,337 @@
+        String content = executeUri(uri, HttpMethod.GET, null);
+        return deserializer.deserialize(clazz, content);
+    }
+
+    public synchronized <T> T executeUri(URI uri, Class<T> clazz, String payload) throws AirParifException {
+        String content = executeUri(uri, HttpMethod.POST, payload);
+        return deserializer.deserialize(clazz, content);
+    }
+
+    @Override
+    public void handleCommand(ChannelUID channelUID, Command command) {
+        logger.debug("The AirParif bridge does not handle commands");
+    }
+
+    private void initiateConnexion() {
+        Version version;
+        try { // This is only intended to validate communication with the server
+            version = executeUri(VERSION_URI, Version.class);
+        } catch (AirParifException e) {
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.COMMUNICATION_ERROR, e.getMessage());
+            return;
+        }
+
+        KeyInfo keyInfo;
+        try { // This validates the api key value
+            keyInfo = executeUri(KEY_INFO_URI, KeyInfo.class);
+        } catch (AirParifException e) {
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.CONFIGURATION_ERROR, e.getMessage());
+            return;
+        }
+
+        thing.setProperty("api-version", version.version());
+        thing.setProperty("key-expiration", keyInfo.expiration().toString());
+        thing.setProperty("scopes", keyInfo.scopes().stream().map(e -> e.name()).collect(Collectors.joining(",")));
+        logger.info("The api key is valid until {}", keyInfo.expiration().toString());
+        updateStatus(ThingStatus.ONLINE);
+
+        ThingUID thingUID = thing.getUID();
+
+        schedule(POLLENS_JOB, () -> updatePollens(new ChannelGroupUID(thingUID, GROUP_POLLENS)), Duration.ofSeconds(1));
+        schedule(AQ_JOB, () -> updateDailyAQBulletin(new ChannelGroupUID(thingUID, GROUP_AQ_BULLETIN),
+                new ChannelGroupUID(thingUID, GROUP_AQ_BULLETIN_TOMORROW)), Duration.ofSeconds(2));
+        schedule(EPISODE_JOB, () -> updateEpisode(new ChannelGroupUID(thingUID, GROUP_DAILY)), Duration.ofSeconds(3));
+    }
+
+    private void updatePollens(ChannelGroupUID pollensGroupUID) {
+        PollensResponse localPollens;
+        try {
+            localPollens = executeUri(POLLENS_URI, PollensResponse.class);
+        } catch (AirParifException e) {
+            logger.warn("Error updating pollens data: {}", e.getMessage());
+            return;
+        }
+
+        updateState(new ChannelUID(pollensGroupUID, CHANNEL_COMMENT), Objects.requireNonNull(
+                localPollens.getComment().map(comment -> (State) new StringType(comment)).orElse(UnDefType.NULL)));
+        updateState(new ChannelUID(pollensGroupUID, CHANNEL_BEGIN_VALIDITY), Objects.requireNonNull(
+                localPollens.getBeginValidity().map(begin -> (State) new DateTimeType(begin)).orElse(UnDefType.NULL)));
```

Comment:
The DateTimeType should be used with an `Instant`.
For reference: https://github.com/openhab/openhab-addons/pull/17725

Reply:
Changed all `ZonedDateTime`s to `Instant`s